src/store-easy-peasy/slices/wallets/slices/myNearWallet/thunks/handleRedirects.ts

```diff
@@ -17,7 +17,8 @@ const getSearchParams = () => {
     return {
       connectAllowed: true,
       returnTo,
-      accountId: searchParams.get("accountId"),
+      accountId:
+        searchParams.get("accountId") || searchParams.get("account_id"),
```

Comment:
@eclipseeer account id, when I checked was returned in account_id param.

Reply:
they might be using different ways of writing account ID based on the endpoint

Reply:
Good to know